src/rustup-dist/src/manifest.rs

```diff
@@ -102,6 +103,9 @@ impl Manifest {
         let packages = Self::packages_to_table(self.packages);
         result.insert("pkg".to_owned(), toml::Value::Table(packages));
 
+        let profiles = Self::profiles_to_table(self.profiles);
+        result.insert("profiles".to_owned(), toml::Value::Table(profiles));
+
```

Comment:
Okay, so this implies that the profile content is dictated by the manifest provided from the compiler team?

Given that, I understand minimal/default/complete to be an API which we'll be guaranteeing to that team.  Do we need to document that somewhere?

Reply:
Not really, the rustup team should probably own the manifest and its generation (maybe with the infra team).